docker/postgres-init-dev.sql

```diff
@@ -0,0 +1,3 @@
+-- Create user for testing read-only connections (DATABASE_URL=postgres://ons_alpha_read_only@db/ons_alpha)
+CREATE ROLE ons_alpha_read_only WITH LOGIN;
+GRANT SELECT ON ALL TABLES IN SCHEMA public to ons_alpha_read_only;
```

Comment:
q: what about renditions? when you try to access a rendition for the first time, it will create a `Rendition` object..

Reply:
Oh, that's a problem. We might need to selectively grant rendition write access, although that potentially introduces some path-traversal issues for S3.

Reply:
Can we not pre-generate them instead of on-demand?

Reply:
Unfortunately not.

As this needs more thinking, I've extracted it out into a separate ticket.